spec/graphql/static_validation/rules/fields_will_merge_spec.rb

```diff
@@ -45,4 +45,27 @@
     ]
     assert_equal(expected_errors, error_messages)
   end
+
+  describe "when checking for directive conflicts" do
+    let(:query_string) {"
+      query getCheese {
+        cheese(id: 1) {
+          name,
+          ...F0,
+          ...F1
+        }
+      }
+      fragment F0 on Cheese {
+        name @include(if: true)
+      }
+      fragment F1 on Cheese {
+        name @skip(if: false)
```

Comment:
`@skip(if: true)` is also valid - any combination of `<no directive>`, `@skip(if: true)`, `@skip(if: false)`, `@include(if: true)`, and `@include(if: false)` should be considered valid, either in combination on the same AST field or split across fragments. This is because `@skip` and `@include` have specific semantics in the GraphQL spec, so we know they can merge cleanly - which is the intention of this `fields_will_merge` validation.